reconcile/jira_permissions_validator.py

```diff
@@ -166,12 +166,16 @@ def board_is_valid(
                 )
                 error |= ValidationError.INVALID_PRIORITY
     except JIRAError as e:
-        if e.status_code != 403:
+        if e.status_code == 401:
+            # sporadic 401 errors, retrying
+            pass
```

Comment:
I'd add at least a debug message in case, at some point this stops being intermittent.

Reply:
Done